tests/kokkos/kokkos_utils_tests.cpp

```diff
@@ -244,4 +245,81 @@ TEST_CASE("parallel_reduce", "[kokkos_utils]")
   test_parallel_reduce<Real,15,false> ();
 }
 
+
+template<typename Scalar, int TotalSize, int VectorSize, bool Serialize>
+void test_view_reduction()
+{
+  using Device = ekat::DefaultDevice;
+  using MemberType = typename ekat::KokkosTypes<Device>::MemberType;
+  using ExeSpace = typename ekat::KokkosTypes<Device>::ExeSpace;
+
+  ekat::ExeSpaceUtils<Serialize,ExeSpace> ExeSpaceUtils; 
+  
+  using PackType = ekat::Pack<Scalar, VectorSize>;
+  using ViewType = Kokkos::View<PackType*,ExeSpace>;
+
+  const int view_length = (int)std::ceil(1.0*TotalSize/VectorSize);
```

Comment:
If you want to compute the number of packs, you can use the `npack` utility from `ekat_pack.hpp`:
```
const int view_length = npack<PackType>(TotalSize);
```